core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -227,6 +228,16 @@ case class SubmissionWorkflowStatusResponse(
   workflowStatus: String,
   count: Int)
 
+case class SubsystemStatus(
+  ok: Boolean,
+  messages: Option[List[String]]
+)
+
+case class StatusCheckResponse(
+  ok: Boolean,
+  systems: Map[Subsystem, SubsystemStatus]
+)
+
```

Comment:
Why are these in `ExecutionModel`? If you put them somewhere in the `model/` folder you can reuse them in Orchestration by republishing the rawlsModel library, too.